docs/quickstart/go.md

```diff
@@ -4,4 +4,186 @@ headline: Go Quick Start
 layout: docs
 sidenav: doc-side-quickstart-nav.html
 type: markdown
----
\ No newline at end of file
+---
+
+<p class="lead">This guide gets you started with gRPC in Go with a simple
+working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+#### Go version
+Make sure that Go installed is version 1.5 or higher.
+
+```sh
+$ go version
+```
+
+For installtion instruction follow this guide: [Getting Started - The Go Programming Language](https://golang.org/doc/install)
+
+#### Install gRPC
+To install gRPC on your system use the following command:
+```sh
+$ go get google.golang.org/grpc
+```
+
+#### Install Protocol Buffers v3
+
+While not mandatory, gRPC usually leverages Protocol Buffers v3 for service
+definitions and data serialization.
```

Comment:
This sentence is a bit wordy/superfluous for this quickstart - users can find out more about how/if gRPC uses protocol buffers in the more detailed guides and tutorials. I'd take it out.